### ISSUE TITLE
Crash report 8.9: NPE in ThemeBrowserActivity.setThemeBrowserFragment

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#6906]
On a Jetpack or wpcom site, when I tap the Themes button, I get this crash:

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android.beta/org.wordpress.android.ui.themes.ThemeBrowserActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void org.wordpress.android.ui.themes.ThemeBrowserActivity.setThemeBrowserFragment(org.wordpress.android.ui.themes.ThemeBrowserFragment)' on a null object reference
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2298)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2360)
	at android.app.ActivityThread.access$800(ActivityThread.java:144)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1278)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:135)
	at android.app.ActivityThread.main(ActivityThread.java:5223)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:899)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:694)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void org.wordpress.android.ui.themes.ThemeBrowserActivity.setThemeBrowserFragment(org.wordpress.android.ui.themes.ThemeBrowserFragment)' on a null object reference
	at org.wordpress.android.ui.themes.ThemeBrowserFragment.onActivityCreated(ThemeBrowserFragment.java:139)
	at android.app.Fragment.performActivityCreated(Fragment.java:2061)
	at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:912)
	at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1067)
	at android.app.BackStackRecord.run(BackStackRecord.java:833)
	at android.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1452)
	at android.app.Activity.performStart(Activity.java:5948)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2261)
	... 10 more
```

**Comment by daniloercoli** [comment]
I can't reproduce the issue here. It's also a weird issue, it seems that the host activity is `null` in `onActivityCreated` of the Fragment.

Android version?

**Comment by maxme** [comment]
I can't repro that one, I don't see any reports in crashlytics in the current beta or alpha. Closing it.

**Comment by maxme** [comment]
This one happens in 8.9 and seems fairly common

```
Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/org.wordpress.android.ui.themes.ThemeBrowserActivity}: java.lang.NullPointerException
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2200)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2249)
       at android.app.ActivityThread.access$800(ActivityThread.java:141)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1212)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:136)
       at android.app.ActivityThread.main(ActivityThread.java:5113)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:796)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:612)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by java.lang.NullPointerException
       at org.wordpress.android.ui.themes.ThemeBrowserFragment.onActivityCreated(ThemeBrowserFragment.java:139)
       at android.app.Fragment.performActivityCreated(Fragment.java:1708)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:908)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1062)
       at android.app.BackStackRecord.run(BackStackRecord.java:690)
       at android.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1449)
       at android.app.Activity.performStart(Activity.java:5251)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2173)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2249)
       at android.app.ActivityThread.access$800(ActivityThread.java:141)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1212)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:136)
       at android.app.ActivityThread.main(ActivityThread.java:5113)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:796)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:612)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

cc @tonyr59h @oguzkocer

**Comment by oguzkocer** [comment]
I'll check this out first thing tomorrow!

**Comment by lethalskillzz** [comment]
Still crashing API 22

> 12-07 15:01:22.013 7836-7836/org.wordpress.android E/AndroidRuntime: FATAL EXCEPTION: main
                                                                     Process: org.wordpress.android, PID: 7836
                                                                     java.lang.RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/org.wordpress.android.ui.themes.ThemeBrowserActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void org.wordpress.android.ui.themes.ThemeBrowserActivity.setThemeBrowserFragment(org.wordpress.android.ui.themes.ThemeBrowserFragment)' on a null object reference
                                                                         at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2348)
                                                                         at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2410)
                                                                         at android.app.ActivityThread.access$800(ActivityThread.java:151)
                                                                         at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1313)
                                                                         at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                         at android.os.Looper.loop(Looper.java:135)
                                                                         at android.app.ActivityThread.main(ActivityThread.java:5348)
                                                                         at java.lang.reflect.Method.invoke(Native Method)
                                                                         at java.lang.reflect.Method.invoke(Method.java:372)
                                                                         at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:947)
                                                                         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:742)
                                                                      Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void org.wordpress.android.ui.themes.ThemeBrowserActivity.setThemeBrowserFragment(org.wordpress.android.ui.themes.ThemeBrowserFragment)' on a null object reference
                                                                         at org.wordpress.android.ui.themes.ThemeBrowserFragment.onActivityCreated(ThemeBrowserFragment.java:139)
                                                                         at android.app.Fragment.performActivityCreated(Fragment.java:2061)
                                                                         at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:912)
                                                                         at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1067)
                                                                         at android.app.BackStackRecord.run(BackStackRecord.java:838)
                                                                         at android.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1452)
                                                                         at android.app.Activity.performStart(Activity.java:6030)
                                                                         at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2311)
                                                                         at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2410) 
                                                                         at android.app.ActivityThread.access$800(ActivityThread.java:151) 
                                                                         at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1313) 
                                                                         at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                         at android.os.Looper.loop(Looper.java:135) 
                                                                         at android.app.ActivityThread.main(ActivityThread.java:5348) 
                                                                         at java.lang.reflect.Method.invoke(Native Method) 
                                                                         at java.lang.reflect.Method.invoke(Method.java:372)

**Comment by oguzkocer** [comment]
@lethalskillzz The fix has not been released yet.

**Comment by oguzkocer** [comment]
Fixed in #6938.
